src/Geopilot.Api/Models/Mandate.cs

```diff
@@ -42,4 +49,60 @@ public class Mandate : IIdentifiable
     /// Data deliveries that have been declared fulfilling the mandate.
     /// </summary>
     public List<Delivery> Deliveries { get; set; } = new List<Delivery>();
+
+    /// <summary>
+    /// Transforms the <see cref="Coordinates"/> list to a <see cref="SpatialExtent"/> polygon.
+    /// </summary>
+    public void SetPolygonFromCoordinates()
+    {
+        if (Coordinates.Count != 2)
+        {
+            SpatialExtent = Geometry.DefaultFactory.CreatePolygon();
+        }
+        else
+        {
+            SpatialExtent = Geometry.DefaultFactory.CreatePolygon(new NetTopologySuite.Geometries.Coordinate[]
+            {
+                new (Coordinates[0].X, Coordinates[0].Y),
+                new (Coordinates[0].X, Coordinates[1].Y),
+                new (Coordinates[1].X, Coordinates[1].Y),
+                new (Coordinates[1].X, Coordinates[0].Y),
+                new (Coordinates[0].X, Coordinates[0].Y),
+            });
+        }
+    }
+
+    /// <summary>
+    /// Transforms the <see cref="SpatialExtent"/> polygon to a <see cref="Coordinates"/> list.
+    /// </summary>
+    /// <exception cref="InvalidOperationException">Exception thrown if the coordinate list is of an invalid length.</exception>
+    public void SetCoordinateListFromPolygon()
+    {
+        Coordinates = new List<Coordinate>();
+
+        switch (SpatialExtent.Coordinates.Length)
+        {
+            case 0:
+                break;
+            case 5:
+                double minX = SpatialExtent.Coordinates[0].X;
+                double minY = SpatialExtent.Coordinates[0].Y;
+                double maxX = SpatialExtent.Coordinates[0].X;
+                double maxY = SpatialExtent.Coordinates[0].Y;
+
+                foreach (var coord in SpatialExtent.Coordinates)
+                {
+                    minX = Math.Min(minX, coord.X);
+                    minY = Math.Min(minY, coord.Y);
+                    maxX = Math.Max(maxX, coord.X);
+                    maxY = Math.Max(maxY, coord.Y);
+                }
+
+                Coordinates.Add(new Coordinate { X = minX, Y = minY });
+                Coordinates.Add(new Coordinate { X = maxX, Y = maxY });
+                break;
+            default:
+                throw new InvalidOperationException($"Unsupported number of coordinates. Spatial extent must be a rectangle.");
```

Comment:
REQ: Unnötigen Operator entfernen
```suggestion
                throw new InvalidOperationException("Unsupported number of coordinates. Spatial extent must be a rectangle.");
```